git_scripts/open_internal_pr.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+### For Dune employees.
+### This script fetches a branch from a forked repo, opens a new internal branch, and opens a pull request
+### Example usage:
+### To open a pull request from 0xbitfly from their master branch
+### run the following command from your terminal in the abstractions repo
```

Comment:
running it from the abstractions repo didn't work for me, I had to run it inside the git_scripts folder